packages/solid-urql/src/createQuery.ts

```diff
@@ -0,0 +1,334 @@
+import {
+  type AnyVariables,
+  type OperationContext,
+  type DocumentInput,
+  type OperationResult,
+  type RequestPolicy,
+  type OperationResultSource,
+  createRequest,
+} from '@urql/core';
+import {
+  batch,
+  createComputed,
+  createMemo,
+  createResource,
+  onCleanup,
+} from 'solid-js';
+import { createStore, unwrap } from 'solid-js/store';
+import { useClient } from './context';
+import { type MaybeAccessor, asAccessor } from '@solid-primitives/utils';
```

Comment:
I don't really see why we'd have to depend on `@solid-primitives/utils` for a single, primitive helper, so I'd much prefer this to be re-implemented/vendored in `@urql/solid` itself, and keep this independent.

Reply:
Sure, extracted relevant code from `@solid-primitives/utils`